generator.js

```diff
@@ -3,6 +3,9 @@ const semver = require("semver");
 module.exports = (api) => {
   const isTs = api.entryFile.endsWith(".ts");
   const { dependencies, name } = require(api.resolve("package.json"));
+  if (!dependencies) {
+    throw Error(`Could not find any dependencies in package.json. Verify that you're in the correct directory, and that dependencies are defined.`);
```

Comment:
It might help to print the full file path to the package.json that we're looking for.